plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1252,4 +1254,16 @@ public HiveConfig setPartitionProjectionEnabled(boolean enabledAthenaPartitionPr
         this.partitionProjectionEnabled = enabledAthenaPartitionProjection;
         return this;
     }
+
+    public S3StorageClassFilter getS3StorageClassFilter()
+    {
+        return s3StorageClassFilter;
+    }
+
```

Comment:
Does `s3.storage-class-filter` property make sense for `S3FileSystemConfig` as well?

https://trino.io/docs/current/object-storage/file-system-s3.html

cc @anusudarsan @electrum

Reply:
@bangtim pls move in any case the configuration to `HiveS3Config` along with the other `hive.s3.*` config properties.